nix/dapptools-src.nix

```diff
@@ -0,0 +1,12 @@
+{ stdenv }:
+
+stdenv.mkDerivation {
+  name = "dapptools-src";
+  src = ../.;
+
+  phases = [ "unpackPhase" "installPhase" ];
```

Comment:
why do we need `unpackPhase`?

Reply:
To move the source to the build directory.